Makefile.in

```diff
@@ -479,19 +479,26 @@ ghwdump: ghwdump$(EXEEXT)
 ghwdump$(EXEEXT): $(GHWDUMP_OBJS)
 	$(CC) -o $@ $(GHWDUMP_OBJS)
 
+ghwlib$(SOEXT): ghwlib.o
```

Comment:
If this was named `libghw$(SOEXT)`, it could be linked against with `-lghw`.

Reply:
Absolutely. And it's not the first time I/we do this exact same mistake... Thanks for catching it!